NDTraining/Sv_Selenium/Test/CreateSavedSearchPageTest.cs

```
@@ -0,0 +1,68 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using Sv_Selenium.PageObjects;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sv_Selenium.Test
+{
+    class CreateSavedSearchPageTest : BaseTest
+    {
+        private LoginPage objLoginPage;
```

this method can be simplified if you use parameterized selector:
create method that returns selector for list view item by its name:
```
private By ListViewItem(string itemName) => By.Xpath($"//span[contains(@class,'lvName-span')][@title='{itemName}']");
```
then here:
```
WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(15));
wait.Until(x => driver.FindElement(ListViewItem(criteria));
)
```
you can either make this method void - it will throw exception if item is not found - or wrap it into try catch and return true/false (and make assert statement in test)